src/sensor/pingsensor.cpp

```diff
@@ -14,6 +14,21 @@ PingSensor::PingSensor()
     connect(dynamic_cast<PingParserExt*>(_parser), &PingParserExt::parseError, this, &PingSensor::parserErrorsUpdate);
 }
 
+void PingSensor::connectLink(const LinkConfiguration conConf, const LinkConfiguration& logConf)
+{
+    Sensor::connectLink(conConf, logConf);
+    if (_parser) {
+        connect(link(), &AbstractLink::newData, this, [this](QByteArray data) {
```

Comment:
You are doing a deep copy of the QByteArray, use reference.

Reply:
nice :eye: